File: Java/프로그래머스/0/120894. 영어가 싫어요/영어가 싫어요.java
class Solution {
    public long solution(String numbers) {
        long answer = 0;
        String result="";
        String answer1="";
        char ch[]=numbers.toCharArray();
        int i=0;
        while(i<ch.length){    
            result+=ch[i];
            i++;
            if(result.equals("zero")) {
                answer1+="0";
                result="";
            }
            else if(result.equals("one")) {
                answer1+="1";
                result="";
            }
            else if(result.equals("two")) {
                answer1+="2";
                result="";
            }
            else if(result.equals("three")) {
                answer1+="3";
                result="";
            }
            else if(result.equals("four")) {
                answer1+="4";
                result="";
            }
            else if(result.equals("five")) {
                answer1+="5";
                result="";
            }
            else if(result.equals("six")) {
                answer1+="6";
                result="";
            }
            else if(result.equals("seven")) {
                answer1+="7";
                result="";
            }
            else if(result.equals("eight")) {
                answer1+="8";
                result="";
            }
            else if(result.equals("nine")) {
                answer1+="9";
                result="";
                }
        }
        answer=Long.parseLong(answer1);
        return answer;
    }
}

File: 프로그래머스/0/120894. 영어가 싫어요/영어가 싫어요.js
function solution(numbers) {
    var answer = numbers.replaceAll("zero", 0);
    answer = answer.replaceAll("one", 1);
    answer = answer.replaceAll("two", 2);
    answer = answer.replaceAll("three", 3);
    answer = answer.replaceAll("four", 4);
    answer = answer.replaceAll("five", 5);
    answer = answer.replaceAll("six", 6);
    answer = answer.replaceAll("seven", 7);
    answer = answer.replaceAll("eight", 8);
    answer = answer.replaceAll("nine", 9);
    return Number(answer);
}